src/features/settings/Settings.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+
+import {
+  useSelector, useDispatch,
+} from "react-redux";
+
+import CheckBox from "@mui/material/Checkbox";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormGroup from "@mui/material/FormGroup";
+
+import {
+  persistDisclaimersStatus, selectDisclaimersStatus,
+} from "./settingsSlice";
+
+export default function Settings() {
+  const { t } = useTranslation("settings");
+  const disclaimersStatus = useSelector(selectDisclaimersStatus);
+
+  const dispatch = useDispatch();
+
+  const handleDisclaimerStateChange = useCallback((e) => {
+    dispatch(persistDisclaimersStatus(e.target.checked));
+  }, [dispatch]);
+
+  const disclaimersCheckboxString = t("disclaimersCheckbox", { state: !disclaimersStatus ? t("enable") : t("disable") } );
+
+  return (
+    <FormGroup>
+      <FormControlLabel
+        control={
+          <CheckBox
```

Comment:
Here it would be nice to use the same ```Switch``` component that we already use on the Startup page, just to be consistent. Also we don't need different text for enable/disable.